packages/bpk-component-calendar/src/BpkCalendarNav-test.js

```diff
@@ -55,6 +55,38 @@ describe('BpkCalendarNav', () => {
     expect(asFragment()).toMatchSnapshot();
   });
 
+  it('should use given previousMonth and nextMonth labels correctly', () => {
```

Comment:
New tests to check that there isn't any regressions in labels, or in the default logic in the future. Other functionality is already covered in the existing tests.